app/helpers/image_service_helper.rb

```diff
@@ -88,6 +88,12 @@ def member_image_tag(parent_id:, member:, size_key: nil, lazy: false)
       }
     }
 
+    if member.respond_to? 'representative_content_type' and
+      !member.representative_content_type.blank? and
+      member.representative_content_type.include? "pdf"
+      args[:class] += ['pdf_border']
```

Comment:
I think this should probably be `== 'application/pdf'` instead of `include? 'pdf'`; I think the latter is vague for no reason.